src/pages/__tests__/Calculator.test.tsx

```diff
@@ -10,4 +10,21 @@ describe('calculator page', () => {
 
     expect(getByText(/constructed a calculator that lets/i)).toBeInTheDocument()
   })
+
+  it('activity details section reveals after location is selected', () => {
+    const { getByText, queryByText, getByRole } = render(<Calculator />, {
+      wrapper: AllProviders,
+    })
+
+    expect(getByText(/constructed a calculator that lets/i)).toBeInTheDocument()
```

Comment:
This is already tested in the previous test; no need to repeat.

Reply:
Oops

Reply:
Fixed.